src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,59 @@
+package duke.task;
+
+public class Task {
+    private String task;
```

Comment:
I am a bit confused. Does the variable "task:String" holds the title of the task? If so, please consider renaming it to something like "taskName" or "taskTitle" for clarification. Otherwise, the Task object would have the method getTask() for this title, which sounds like returning the object itself.